library/agent/protect.ts

```diff
@@ -132,6 +134,8 @@ function getWrappers() {
     new BetterSQLite3(),
     new Postgresjs(),
     new Fastify(),
+    new Koa(),
+    new KoaRouter(),
```

Comment:
I wouldn't mind these being one sink but up to you!

Reply:
Because the koa router already consists of two different packages to wrap, it's maybe a bit better readable if it's splitted. Maybe there are more koa routers to add in the future.